internal/cmdliveinit/cmdliveinit.go

```diff
@@ -149,22 +164,12 @@ func (c *ConfigureInventoryInfo) Run(ctx context.Context) error {
 	} else {
 		name = c.Name
 	}
-	// Generate the inventory id if one is not specified through a flag.
-	if c.InventoryID == "" {
```

Comment:
What happens if the user doesn't specify an InventoryID through the command line?

Reply:
From the design doc, if the user does not provide an `inventoryID`, we do not write the `inventoryID` to the resourcegroup yaml file. On the cluster, all resources managed by kpt will contain the `cli-utils.sigs.k8s.io/inventory-id` label with an empty default value. In essence, we mask `inventoryID` as an implementation detail to the user if one is not explicitly provided.